.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 types
-dist
\ No newline at end of file
+dist
+.vscode
```

Comment:
We need to explicitly remove the added ` .vscode/settings.json` from git (this PR) too.

Reply:
I've done that now (just realised I can push to your PR branch directly 😄 )